docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
@fowlslegs I'll let you make a final call. Seems like there is no better option than "Debian Sid (May 10, 2016)" in which case, go for it.